### PR TITLE
Cookiecutter defaults to main

### PR DESCRIPTION
**Pull request by Lnaden**
The PR at https://github.com/MolSSI/cookiecutter-cms/pull/157 updates the cookiecutter to set the default branch to `main` instead of `master`.

The PR here updates the `github` lesson to reflect that.

Please delete the text below before submitting your contribution.

### CODE REVIEW

**Comment by Lnaden** [comment]
image

This is what the block looks like since im using a callout + warning block.

File: _episodes/03-github.md [hunk 1]
```diff
@@ -56,8 +56,7 @@ You should see no output.
 Now, follow the instructions on GitHub under "...or push an existing repository from the command line"
 ~~~
 $ git remote add origin git@github.com:YOUR_GITHUB_USERNAME/molecool.git
-$ git branch -M main
-$ git push -u origin main
+$ git branch
```
Comment (janash) [inline _episodes/03-github.md:59]: I think we will want to leave original instructions here to match instructions on GitHub. If your branch is already named `main`, the command does nothing, so there is no harm in leaving it in.

**Comment by Lnaden** [comment]
If thats the case, this PR can be closed without merge.

**Comment by sjayellis** [comment]
I also think it should be fine to leave the original instructions. Yes, we suggest the cookiecutter for going through the workshop material, but the online resources are also there as reference material for anyone who needs them. It may still be useful to someone who is reviewing a git repository not setup through the cookiecutter.

**Pull request closed without merging**